lib/ansible/module_utils/openstack.py

```diff
@@ -28,6 +28,7 @@
 
 import os
 
+from ansible.compat.six import iteritems
```

Comment:
this should import from because module_utils will be transfered to remote target
`from ansible.module_utils.six import iteritems`